examples/jsm/loaders/GLTFLoader.js

```diff
@@ -2627,6 +2627,8 @@ class GLTFParser {
 
 			} ) ).then( function () {
 
+				result.scene.updateMatrixWorld();
```

Comment:
Note that a glTF file may contain multiple scenes. Perhaps instead:

```suggestion
				for ( const scene of result.scenes ) {

					scene.updateMatrixWorld();

				}
```

Reply:
Sounds good! I've directly committed your suggestion!